Makefile

```
@@ -205,7 +194,7 @@ postgres-gis-pgimg-docker: postgres-gis-pgimg-build
 
 # ----- Special case pg-based image (postgres-ha) -----
 # Special case args: BACKREST_VER, PATRONI_VER
-postgres-ha-pgimg-build: cc-pg-base-image commands $(CCPROOT)/build/postgres-ha/Dockerfile
+postgres-ha-pgimg-build: postgres commands $(CCPROOT)/build/postgres-ha/Dockerfile
```

Maybe this should be:
```suggestion
postgres-ha-pgimg-build: postgres-pgimg-build commands $(CCPROOT)/build/postgres-ha/Dockerfile
```